com.civicrm.hrjobroles/phpunit.xml.dist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<phpunit backupGlobals="false" backupStaticAttributes="false" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" processIsolation="false" stopOnFailure="false" syntaxCheck="false" bootstrap="tests/phpunit/bootstrap.php">
+  <testsuites>
+    <testsuite name="Job roles extension test suit">
```

Comment:
There's a typo in suit (it should be suite). Also, for consistency with other phpunit.xml files, can you please make all first letters uppercase (Job Roles Test Suite)?